components/automate-es-gateway/habitat/config/nginx.conf

```diff
@@ -24,15 +24,15 @@ http {
   gzip_proxied {{cfg.ngx.http.gzip_proxied}};
   gzip_types {{cfg.ngx.http.gzip_types}};
 
-  {{~#if cfg.ngx.main.resolvers}}
-  resolver {{cfg.ngx.main.resolvers}};
-  {{/if~}}
+  {{#if cfg.ngx.main.resolvers.nameservers}}
```

Comment:
Might it be better to only reference the value that is actually required, `nameservers_string`?

Reply:
I was sceptical if `nameservers_string` being a string will also pass for empty string. But as discussed and as per the handlebarjs document, empty string would be inferred as false